api/envoy/config/core/v3/protocol.proto

```diff
@@ -260,6 +260,15 @@ message HttpProtocolOptions {
   // a 431 response for HTTP/1.x and cause a stream reset for HTTP/2.
   google.protobuf.UInt32Value max_headers_count = 2 [(validate.rules).uint32 = {gte: 1}];
 
+  // The maximum combined size of headers.
+  // If unconfigured, the default is 60 KiB, except for HTTP/1 response headers which have a default
+  // of 80KiB.
+  // Requests that exceed this limit will receive a 431 response.
```

Comment:
what happens to responses which exceed this limit? 503?

Reply:
documented